dnf/yum/rpmtrans.py

```diff
@@ -113,7 +113,10 @@ def progress(self, package, action, ti_done, ti_total, ts_done, ts_total):
         pass
 
     def scriptout(self, msgs):
-        """msgs is the messages that were output (if any)."""
+        """Print scriptlet output.
+
+        :param msgs: the scriptlet output
+        """
```

Comment:
I'd prefer the docstring here to contain a note about this being a hook you can overload later. It may be obvious, but on the other hand the docstring says it prints scriptlet output, but the body is just `pass`, so that's confusing at first glance.

Reply:
Tried to clarify the message.